website/src/_posts/2016-09-0.10.md

```diff
@@ -15,7 +15,7 @@ We have done a lot of research. We created [prototypes of Uppy React components]
 
 <!-- more -->
 
-Finally, we are also discussing using [Redux](https://github.com/reactjs/redux) as the internal state management store within Uppy.  You can see some drafts under the [`src/experimental` folder](https://github.com/transloadit/uppy/tree/master/src/experimental).
+Finally, we are also discussing using [Redux](https://github.com/reactjs/redux) as the internal state management store within Uppy.  You can see some drafts under the `src/experimental` folder.
```

Comment:
If we are removing broken links from older blog post, I think we should add a note explaining it no longer exists 
```suggestion
Finally, we are also discussing using [Redux](https://github.com/reactjs/redux) as the internal state management store within Uppy.  You can see some drafts under the `src/experimental` folder (EDIT: this folder has since been removed).
```

Reply:
Might make more sense to create a permalink to when it still existed, if we are explicitly talking about the reference.

Reply:
> Might make more sense to create a permalink to when it still existed, if we are explicitly talking about the reference.

Not very familiar with permalinks, but I did some research and it says that permalinks are seen to be good blog post etiquette, so maybe creating a permalink would be better? (not sure how to create one though)

Reply:
I tried finding it by going back in time through tags.

<img width="653" alt="Screenshot 2021-09-06 at 11 55 12" src="https://user-images.githubusercontent.com/9060226/132198724-a428d1b0-3724-46d0-b0ab-d46b46d046b6.png">

But couldn't find it. So we can just remove this (with an optional note)

Reply:
Ah okay, I'll remove the link and keep the note as well, as suggested.